templates/zig.tpl

```diff
@@ -0,0 +1,21 @@
+!!cnh {"suffix":"zig"}
+/*
```

Comment:
May you please replace the multiline /* ... */ to single-line comment //? As per following information:

Zig does not have traditional multiline comments like /* ... */ found in languages like C. This design choice is intentional and allows for simpler tokenization of each line of code.
Instead, Zig uses single-line comments that begin with // and extend to the end of the line. To create what functions as a "multiline comment," you simply place // at the beginning of each line you wish to comment out.